### PR TITLE
add ios pictures import doc

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/how-to/pictures.en.md [hunk 1]
```diff
@@ -103,3 +102,17 @@ as name.
...
+
+Use iTunes' File Sharing function to import into the QField root folder.
+
+1. Open the iTunes app and click on the iPhone button near the top-left of the iTunes window.
+2. Click on the *File Sharing* option in the left sidebar.
+3. Select the app QField and click on *Add File*. This will open the file browser.
+4. Select the file.
+
+!!! note
+    It is only possible with a PNG raster with a PNW world file at the moment.
```
Comment (m-kuhn) [inline documentation/how-to/pictures.en.md:116]: This sounds like georeferenced raster files, whereas the rest of this page is about pictyre attachments. Is it me misunderstanding something?

Comment (lucienicolier) [reply documentation/how-to/pictures.en.md:116]: > This sounds like georeferenced raster files, whereas the rest of this page is about pictyre attachments. Is it me misunderstanding something?

You are right

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 2]
```diff
@@ -93,6 +93,11 @@ Layer
     layer like a WMS or to take a working copy of an unsupported format
     like an ECW or MrSID layer.
 
+!!! note
+    It is only possible with a PNG raster with a PNW world file at the moment.
```
Comment (m-kuhn) [inline documentation/get-started/tutorials/get-started-qfs.en.md:97]: Is this iOS specific (as the pull request title suggests)? If yes, I think this needs to be mentioned.

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 3]
```diff
@@ -93,6 +93,11 @@ Layer
     layer like a WMS or to take a working copy of an unsupported format
     like an ECW or MrSID layer.
 
+!!! note
+    On iOS devices, It is only possible with a PNG raster with a PNW world file at the moment.
+
+    JPG raster are not supported.
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfs.en.md:99]:
```suggestion
    On iOS devices, it is only possible with a PNG raster with a PNW world file at the moment.

    JPG raster are not supported.
```

Fix `It` should not start with capital letter.

Comment (suricactus) [reply documentation/get-started/tutorials/get-started-qfs.en.md:99]: While I do not have iOS device and cannot test myself, I think it is not that only possible with PNG, but it's ONLY NOT possible with JPG.

Also, I think we can note that single file tiled file formats should be configured with non-jpg tiles too, for example `mbtiles`.

**Review by m-kuhn — APPROVED** [review]

File: documentation/get-started/tutorials/get-started-qfs.en.md (hunk 3 above)
Comment (m-kuhn) [reply documentation/get-started/tutorials/get-started-qfs.en.md]: @nirvn

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 4]
```diff
@@ -93,6 +93,9 @@ Layer
     layer like a WMS or to take a working copy of an unsupported format
     like an ECW or MrSID layer.
 
+!!! note
+    On iOS devices JPEG raster files are not supported.
```
Comment (nirvn) [inline documentation/get-started/tutorials/get-started-qfs.en.md:97]:
```suggestion
    On iOS devices JPEG raster files are not currently supported.
```

**Pull request merged**